src/vscode-dts/vscode.d.ts

```diff
@@ -641,13 +641,22 @@ declare module 'vscode' {
 		/**
 		 * The size in spaces a tab takes. This is used for two purposes:
 		 *  - the rendering width of a tab character;
-		 *  - the number of spaces to insert when {@link TextEditorOptions.insertSpaces insertSpaces} is true.
+		 *  - the number of spaces to insert when {@link TextEditorOptions.insertSpaces insertSpaces} is true
+		 *    and `indentSize` is set to `"tab"`.
```

Comment:
Should that not be 
```suggestion
		 *    and `indentSize` is set to `"tabSize"`.
```
?